src/Illuminate/Foundation/Console/Kernel.php

```diff
@@ -99,12 +99,10 @@ public function __construct(Application $app, Dispatcher $events)
     protected function defineConsoleSchedule()
     {
         $this->app->singleton(Schedule::class, function ($app) {
-            return new Schedule($this->scheduleTimezone());
+            return tap(new Schedule($this->scheduleTimezone()), function ($schedule) {
+                $this->schedule($schedule);
```

Comment:
I don't understand why this was moved into the container's resolution logic of this object? It feels weird to call some other method when simply asking for the object from the container?

Reply:
It need only on Schedule resolving. In other ways no need to call this method. (maybe I misunderstood you)